site/content/docs/02-template-syntax.md

```diff
@@ -1328,6 +1328,28 @@ Named slots allow consumers to target specific areas. They can also have fallbac
 </Widget>
 ```
 
+Components can be placed in a named slot using the syntax `<Component slot="name" />`.
+In order to place content in a slot without losing the flow layout (ie: without using a wrapper element) you can use the special element `<svelte:fragment>`.
```

Comment:
```suggestion
In order to place content in a slot without using a wrapper element, you can use the special element `<svelte:fragment>`.
```

The explanation in parenthesis is clearer, removing the need to the additional elaboration.